packages/TelegramClient-UI.package/TCUChatMessageList.class/instance/isViewingNewestMessage.st

```diff
@@ -0,0 +1,4 @@
+messages
+isViewingNewestMessage
+
+	^ self vScrollBar maximumValue = self vScrollBar value
```

Comment:
Are you sure this works? I believe this only tests if the newest message was fully visible. Is that what you wanted to test?

Reply:
Yeah the intention was to have a boolean to see whether the user would like the chat to automatically scroll to every new message. From my intention this is only the case if the chat is fully scrolled down and as soon as the user scrolls up a little bit he might search for something at that place.
I am not sure if the name states the intention correctly... Do you think the method name is ok or do you think something like `fullyScrolledDown` would be better?

Reply:
I personally like `fullyScrolledDown` better :)

Reply:
maybe isFullyScrolledDown?